x/observer/keeper/msg_server_vote_blame.go

```diff
@@ -13,52 +13,45 @@ import (
 
 func (k msgServer) VoteBlame(
 	goCtx context.Context,
-	vote *types.MsgVoteBlame,
+	msg *types.MsgVoteBlame,
 ) (*types.MsgVoteBlameResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	observationType := types.ObservationType_TSSKeySign
 
 	// GetChainFromChainID makes sure we are getting only supported chains , if a chain support has been turned on using gov proposal, this function returns nil
-	observationChain, found := k.GetSupportedChainFromChainID(ctx, vote.ChainId)
+	observationChain, found := k.GetSupportedChainFromChainID(ctx, msg.ChainId)
 	if !found {
-		return nil, cosmoserrors.Wrap(
+		return nil, sdkerrors.Wrap(
```

Comment:
NIR : we can use `Wrapf`

Reply:
Makes sense, I just deprecated the use of Sprintf where possible.